sites/public/lib/helpers.tsx

```diff
@@ -32,6 +32,16 @@ export const getGenericAddress = (bloomAddress: Address) => {
     : null
 }
 
+export const disableContactFormOption = (id: string, noPhone: boolean, noEmail: boolean) => {
+  if ((id === "phone" || id === "text") && noPhone === true) {
+    return true
```

Comment:
I think instead of doing the `noPhone === true` you could just return noPhone here, same for `noEmail` below

this is just a preference but: instead of writing it as an `else` you could return false since if either of the other conditions were met we would have already returned out